apprentice_teacher.rb

```diff
@@ -1,39 +1,46 @@
-class ApprenticeTeacher
+require_relative 'staff'
+
+class ApprenticeTeacher < Staff
```

Comment:
This says that an apprentice teacher is a staff.  That sounds a little weird -- Staff is the word for all members of the staff.  Maybe `StaffMember`? Or just `Teacher`?